src/FileReaders/NCFileReader.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "FileReader.hpp"
+
+#include "../VariableDesc.hpp"
+
+#include <filesystem>
+#include <iostream>
+
+#include <ncFile.h>
+
+namespace tsm {
+
+class NCFileReader : public FileReader<NCFileReader> {
+
+public:
+    ///
+    NCFileReader(const std::filesystem::path& path);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'NCFileReader' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/app/htmlboss/netcdf-timestamp-mapper/pullRequest?prid=4555540)